src/index.ts

```diff
@@ -237,7 +238,7 @@ export type HttpConnectorConfig = {
 /**
  * @ignore
  */
-const bodyLimit = '20mb'
+export const bodyLimit = '20mb'
```

Comment:
If I read it correctly, this is the _request_ body parser limit, right?
Why is it 20MB? I thought 4MB was the event size maximum.

Reply:
it is not so easy. This is an Actyx best practice value. 
4MB compressed size. + header and other stuff In the emitter.

I think 20 MB is an upper bound with some wired background. :see_no_evil: